openshift/default-catalog-consistency/test/validate/suite_test.go

```diff
@@ -31,7 +33,7 @@ var _ = Describe("Check Catalog Consistency", func() {
 			ctx := context.Background()
 			By(fmt.Sprintf("Validating image: %s", url))
 
-			extractedImage, err := extract.UnpackImage(ctx, url, name)
+			extractedImage, err := extract.UnpackImage(ctx, url, name, authPath)
```

Comment:
This PR as is solves my immediate concern, but I do have a nit suggestion to make it even better:

I'd suggest passing the entire `types.SystemContext` struct into `UnpackImage` rather than just `authFile`. That way, if we ever need to configure any other field on that struct, we don't have to change the `UnpackImage` signature again (and again).